runWPT.sh

```diff
@@ -34,15 +34,22 @@ readonly WPT_BROWSER_PATH="${WPT_BROWSER_PATH:-/Applications/Google Chrome Dev.a
 # The path to the WPT report file.
 readonly WPT_REPORT="${WPT_REPORT:-wptreport.json}"
 
-# The path to the WPT metadata directory.
+# Only set WPT_METADATA if it's not already set.
+if [ -z ${WPT_METADATA+x} ]; then
```

Comment:
Can you give a little bit of inside on why this change is made? I have no context.

Reply:
This line allows providing a custom metadata folder, which is very useful, if you want to re-baseline the tests.